data/scenario_patches/2ca68df.parkpatch

```diff
@@ -35,5 +35,20 @@
                 [ 47, 54 ], [ 47, 55 ], [ 47, 56 ], [ 47, 57 ]
             ]
         }
+    ],
+    "elements_to_delete": [
+        {
+            "element_index": 1,
```

Comment:
Would it make sense to specify the element type instead of its index? You can't have two trees on one tile, after all, but it seems safer than assuming the index, somehow.

Reply:
I don't know, but actually on botany breakers that are two bushes on the same tile, hence the duplicate coordinate. I'll leave this as is for now.